tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "allowJs": true,
```

Comment:
The problem is you can not use `declaration: true` with `allowJS: true` thats why I removed it form the configuration.

```
Compiling with TypeScript
error TS5053: Option 'allowJs' cannot be specified with option 'declaration'.
```

Thanks for the hint I checked the result the `http` folder is missing in the build. There are already open issues for solving this problem: https://github.com/Microsoft/TypeScript/issues/7546

As alternative would you be ok to switch completely to TS? Cause for the file you described there is already an external `.d.ts` file. Wouldn't it much easier to maintain when both files would be combined? Beside that there are only 3 other JS files left (except the tests).